### DISCUSSION TITLE
How to set resources.limits.request.memory in jaeger-agent sidecar ?

### DISCUSSION THREAD

**Discussion by frank-cq** [jaegertracing/jaeger#3606] [Q&A] [answered]
I use annotation sidecar.jaegertracing.io/inject to enable jaeger-client autoinject in my Deployment. However the hpa told me it was failed to get the memory request of the jaeger-client while missing resources.limits.request.memory . Where can I set this configuration without modifying my deployment directly ?

**Comment by frank-cq** [comment, marked as answer]
I cannot close this discussion. I'll send this discussion to jaeger-operator discussion-1832.

**Comment by pavolloffay** [comment]
It's better to ask maintainers to transfer the discussion to other repo or at least cross link them https://github.com/jaegertracing/jaeger-operator/discussions/1832
